website/community/_index.md

```diff
@@ -25,3 +25,6 @@ The recordings are published on the [Gardener Project YouTube channel](https://w
 <p style="margin-top:30px"><strong>Google Calendar</strong></p>
 <iframe src="https://calendar.google.com/calendar/embed?src=gardener.cloud.community%40gmail.com&ctz=Europe%2FBerlin" style="border: 0;margin-bottom:30px" width="800" height="600" frameborder="0" scrolling="no"></iframe>
 
+### Presenting a Topic
+
+If there is a topic you would like to present, message us in our slack channel or get in touch with [Jessica Katz](mailto:jessica.katz@sap.com).
```

Comment:
message us in our slack channel -> shall we name the slack channel  (e.g. \#gardener) and make it a link: https://kubernetes.slack.com/archives/CB57N0BFG? Or this info is already in the page?

Reply:
![Screenshot 2022-10-19 144255](https://user-images.githubusercontent.com/25197046/196681557-bd114016-bce7-4b6f-900d-c331b3ee3b3e.png)
The link to the Gardener slack channel would be right below the content, which is why I decided not to include it. It won't be an issue if we add it, though.